File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file038.f90
program DRB064_outeronly2_orig_no
    use omp_lib
    implicit none

    call foo()
contains
    subroutine foo()
        integer :: i, j, n, m, len
        real, dimension(:,:), allocatable :: b

        len = 100
        allocate (b(len,len))
        n = len
        m = len

        !$omp parallel do private(j)
        do i = 1, n
            do j = 2, m
                b(i,j) = b(i,j-1)
            end do
        end do
        !$omp end parallel do
    end subroutine foo
end program

File: data/gpt-4o/readable_format_HPEC/cpp/file038.cpp
#include <stdio.h>
float b[100][100];
void foo()
{
 int i,j,n,m,len;
 len=100;
 n=len;
 m=len;
#pragma omp parallel for private(j)
 for (i=0;i<n;i++)
 for (j=1;j<m;j++)
 b[i][j]=b[i][j-1];
}
int main()
{
 foo();
 return 0;
}
